_posts/2022-12-14-OLS-Fellowships.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "OLS Fellowships: Announcing our next generation of community changemakers"
+authors: 
+- yochannah
+- pherterich
+- msundukova
+image: https://images.unsplash.com/photo-1516381548400-349d680edb56?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=735&q=80
+photos:
+  name: Photo by Jon Tyson on Unsplash
+  url: https://unsplash.com/photos/r9T0LZv8xWQ
+---
+
+# OLS Fellowships: Announcing our next generation of community changemakers
+
+In September 2022, OLS had a leadership retreat in Scotland, where all four directors gathered together to discuss future plans and strategic director for our organisation. One of the key activities we identified was a desire to work with community members in a more official capacity, beyond the mentorship program that is our core. As a young organisation with funding, we have a relatively high level of flexibility when it comes to defining our mission and how we deliver it. 
+
+## A plan: let’s recruit “Resident Fellows”
+
+Out of these discussions, we developed a new plan: to launch a program that would allow valued community members to spend paid time working with OLS, but also have paid time to explore things that interest them - whether that be research, community management, coaching, consultancy, or anything else that is OLS-aligned. 
+
+Resident fellows are set a fixed budget and employed (if in the UK) or offered a consultant/contractor position, if outside the UK. This budget flexes to suit the fellow - it could be a short-term full time position, or a longer-term part time position of up to 12 months - so long as either way, up to 50% of the time is on OLS-specific tasks specified by the directors or community. Some of the budget can be spent directly (e.g. on books or training or software) rather than used to pay for work time. 
+
+Fellows are encouraged to use their creativity to explore what interests them, and are welcome to apply for grant or contract funding as a PI working under the OLS umbrella. 
+
+
+## Meet the fellows
+
+### Patricia Herterich
```

Comment:
Do you want to do as in Paz post with pictures of you on the side? It will make the post nicer to read, no?

Reply:
@msundukova do you want to have a picture added? If so, which one?